app/src/main/java/com/teamagam/gimelgimel/app/utils/NetworkUtil.java

```diff
@@ -96,7 +98,8 @@ public static InetAddress getIp() {
         return sIp;
     }
 
-    public static String getMac() {
-        return sMacAddress;
+    public static String getUserName(Context context) {
```

Comment:
See the comment below before reading...

This method is currently at NetowrkUtils, which makes no sense.
If all of the MAC logic is removed as I suggested at the comment below, the util class can be renamed. 
Otherwise, It should be moved to more suiting place !resolve